src/Http/Controllers/Controller.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Solutionplus\Payable\Http\Controllers;
+
+use Illuminate\Foundation\Bus\DispatchesJobs;
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Foundation\Validation\ValidatesRequests;
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+
+class Controller extends BaseController
```

Comment:
**Blocking**

There is no need for this file while we don't have a real controller in the package.

in stub control we still need to extend controller class which will exists in Laravel normal path. So, after publishing the stub file it will be ready to use as a normal controller